how-to/migrate-from-a-previous-version/README.md

```diff
@@ -0,0 +1,107 @@
+<img src="../../assets/OpenFin-Workspace-Starter.png" width="100%" alt="OpenFin Workspace Example Application -- Adding your application to Storefront" />
+
+OpenFin Workspace is currently **only supported on Windows**.
+
+# Migrate from a previous version - From v1-v3 to v4
+
+With Workspace 4.0, OpenFin has introduced the ability for Workspace customers to have  more granular control of their Workspace implementation. This control is enabled through Workspace by exposing an API that allows for Provider Apps to perform the function of a CLI Provider. This approach allows Provider Apps to register with the Home API and then perform such actions as:
+
+Manage the application, view and workspaces content available in Workspace
+Provide an async (aka “lazy”) Search
+Provide in-memory auto-complete Search
+Display icons for all registered Providers in the Home/Search UI 
+Selecting a given CLI provider icon to show the results from that provider
+Apply icons/ logos 
+
+With the addition of the CLI Provider concept, OpenFin has deprecated Workspace Desktop Owner Setting overrides (customConfig options ) that pertain to apps and workspaces REST URLs in favor of this programmatic API approach.
+
+## Behavior Changes
+* The fins link or desktop icon to start workspace components is no longer supported. These must be started by a workspace platform or CLI provider. The system will now return an error if the fins link is used to start a workspace component.
+* The “/W” entry point into the Workspaces directory has been removed. 
+* The command buttons (Storefront, Notifications etc) no longer show on the Home UI. They are available as root commands by typing /. 
+* You can no longer specify a Content Discovery Service (CDS). An example is available for how to mimic this functionality using the new APIs available in the howto.
+* Home and Store are now enabled by default. However, they will not show unless a platform has registered with them.
+* Workspace service is no longer supported. Any saved workspaces are only saved locally. Additional workspace related APIs will be released in upcoming releases.
+* Unlike the v1 search API, you must select a CLI provider to see the results from that provider. However, if only one provider is registered then it is automatically selected.
+
+## I used DesktopOwnerSettings to configure the logo. How do I do that now?
```

Comment:
Richard Gutan told me on Wednesday (Nov-24) morning that `style.iconUrl` is still in place.